integration-tests/scenarios/grantDelegation.test.ts

```diff
@@ -1,30 +1,63 @@
 import "@frequency-chain/api-augment";
-import { ApiRx } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { Codec } from "@polkadot/types/types";
+import { u16, u64 } from "@polkadot/types";
 import assert from "assert";
-import { connect } from "../scaffolding/apiConnection";
-import { createMsa, createProvider, createSchema, grantDelegation, grantSchemaPermissions, revokeDelegationByDelegator } from "../scaffolding/extrinsicHelpers";
-import { AccountFundingInputs, createAndFundAccount, generateFundingInputs, txAccountingHook, signPayloadSr25519 } from "../scaffolding/helpers";
+import { AddProviderPayload, Extrinsic, ExtrinsicHelper } from "../scaffolding/extrinsicHelpers";
+import { createAndFundKeypair, createKeys, generateDelegationPayload, signPayloadSr25519 } from "../scaffolding/helpers";
 
 describe("Delegation Scenario Tests", function () {
-    this.timeout(15000);
-
-    let fundingInputs: AccountFundingInputs;
-
-    let api: ApiRx;
     let keys: KeyringPair;
-    let schemaId: Codec;
+    let otherMsaKeys: KeyringPair;
+    let noMsaKeys: KeyringPair;
+    let providerKeys: KeyringPair;
+    let otherProviderKeys: KeyringPair;
+    let schemaId: u16;
+    let providerId: u64;
```

Comment:
If these are MessageSourceIds, the naming should match `msaId` and `otherMsaId` because that's what they are, and what people will be searching on when looking at changes/debugging.

Reply:
I don't see the concept of `provider ID` referenced in the spec anywhere. You must create an MSA first, then create a Provider. So presumably `providerId`, though the same format (`u64`) as an `msaId`, is not the same thing.

Reply:
No, it's not referenced in the spec, because it's a [convenience wrapper for a MessageSourceId](https://github.com/LibertyDSNP/frequency/blob/main/common/primitives/src/msa.rs#L95), like DelegatorId.  Only by a technicality is it not an MSA Id, but your argument is fair enough, and it's just a nitpick anyway.